# message 42
# novatel_msgs/CommonHeader header

string position_type
